tests/test_funcs.py

```diff
@@ -164,12 +172,101 @@ def test_parent():
     assert parent.z == 9
 
 
+def test_parent_multi():
+    parent = mercantile.parent(486, 332, 10, zoom=8)
+    assert parent == (121, 83, 8)
+    assert parent.z == 8
+
+
 def test_children():
     children = mercantile.children(243, 166, 9)
     assert len(children) == 4
     assert (486, 332, 10) in children
 
 
+def test_children_multi():
+    children = mercantile.children(243, 166, 9, zoom=11)
+    assert len(children) == 16
+    targets = [
+        (972, 664, 11),
+        (973, 664, 11),
+        (973, 665, 11),
+        (972, 665, 11),
+        (974, 664, 11),
+        (975, 664, 11),
+        (975, 665, 11),
+        (974, 665, 11),
+        (974, 666, 11),
+        (975, 666, 11),
+        (975, 667, 11),
+        (974, 667, 11),
+        (972, 666, 11),
+        (973, 666, 11),
+        (973, 667, 11),
+        (972, 667, 11),
+    ]
+    for target in targets:
+        assert target in children
+
+def test_children_parent_bad_tile_args():
+    try:
+        mercantile.children((243, 166, 9), 10)
+    except ValueError as e:
+        assert "Could not parse tile!" in str(e)
+
+    try:
+        mercantile.parent((243, 166, 9), 1)
+    except ValueError as e:
+        assert "Could not parse tile!" in str(e)
```

Comment:
Two things: 

1. this should be two test functions, not two tests within one function. Pytest's parametrize is what we should use.
2. We prefer `with pytest.raises(ValueError)` to explicitly trying and catching exceptions.